book/src/concepts/entity_and_component.md

```diff
@@ -0,0 +1,105 @@
+# Entity and Component
+
+## What are `Entity` and `Component` ?
+
+An `Entity` conceptually represents a single object in your world. `Component` represents one aspect of an object. For example, a bottle of water has a shape, a volume, a color and is made of a material (usually plastic). In this example, the bottle is the entity, and the properties are components.
+
+## Entity and Component in Amethyst
+In an inheritance design, entity usually contains components. All the data and methods related to an entity are stored within. However, in the ECS design, entity is just a general purpose object. In fact, the implementation of `Entity` in Amethyst is simply:
+```rust,no_run,noplaypen,ignore
+struct Entity(u32, Generation);
+```
+where u32 is the id of the entity and generation is used to check if the entity has been deleted.
+
+`Entity`s are stored in a special container `EntitiesRes`. Whereas the data associated with the entities are grouped into components and stored in the designated storages.
+
+Consider an example where you have three objects: two bottles and a person.
+
+* Bottle A: x: 150.0, y: 202.1, shape: "round", color: "red"
+
+* Bottle B: x: 15.0, y: 22.0, shape: "square", color: "white"
+
+* Person C: x: 100.5, y: 300.8, name: "Peter"
```

Comment:
Hm, I'm quite pedantic about how item attributes are presented &ndash; I'd like this to be aligned somehow. Let's see.

Code

```
Bottle A: x: 150.0, y: 202.1, shape: "round",  color: "red"
Bottle B: x:  15.0, y:  22.0, shape: "square", color: "white"
Person C: x: 100.5, y: 300.8, name: "Peter"
```

Table

| Item     | x     | y     | shape    | color   | name    |
| ---      | ---:  | ---:  | ---      | ---     | ---     |
| Bottle A | 150.0 | 202.1 | "round"  | "red"   |         |
| Bottle B |  15.0 |  22.0 | "square" | "white" |         |
| Person C | 100.5 | 300.8 |          |         | "Peter" |

I think I like table, could we go with that?

Reply:
Table looks great! will do it!